src/material-examples/expansion-steps/expansion-steps-example.ts

```diff
@@ -0,0 +1,22 @@
+import {Component} from '@angular/core';
+
+
+@Component({
+  selector: 'expansion-overview-example',
+  templateUrl: 'expansion-overview-example.html',
+})
```

Comment:
The template and selector need to change too! Looks like copy/paste from the other example